opensearch/bbuy_queries.json

```diff
@@ -1,4 +1,32 @@
 {
+  "mappings": {
+    "properties": {
+      "user": {
+        "type": "keyword"
+      },
+      "sku": {
+        "type": "keyword"
+      },
+      "category": {
+        "type": "keyword"
+      },
+      "query": {
+        "type": "text",
+        "fields": {
+          "keyword": {
+            "type": "keyword"
+          }
+        },
+        "analyzer": "english"
+      },
+      "click_time": {
+        "type": "keyword"
```

Comment:
Having `keyword` type analyzers for date fields will not allow using them for more advanced date queries such as range etc.